canonical-websites/www.ubuntu.com
# update /desktop/education to vbt

## Done

* update /desktop/education to vbt

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/desktop/education)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/5.%20Desktop?preview=desktop-education.png) and the [demo](http://www.ubuntu.com-1657-desktop-education.demo.haus/desktop/education)

## Issue / Card

Fixes #1657
